nodes/cma/nodes-cma-autoscaling-custom-trigger-auth.adoc

```diff
@@ -15,7 +15,21 @@ Alternatively, to share credentials between objects in multiple namespaces, you
 
 Trigger authentications and cluster trigger authentication use the same configuration. However, a cluster trigger authentication requires an additional `kind` parameter in the authentication reference of the scaled object.
 
-.Example trigger authentication with a secret
+.Example secret for Basic Authentication
```

Comment:
```suggestion
.Example secret for basic authentication
```
I would think all of these would have no capitalization for "basic authentication".

Reply:
@kcarmichael08 I actually got this one from a Vale check. Apparently, [Basic authentication is correct](https://redhat-documentation.github.io/supplementary-style-guide/#basic-http-authentication).